src/locale/locales/de/messages.po

```diff
@@ -301,11 +275,11 @@ msgstr "App-Passwörter"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:250
 msgid "Appeal content warning"
-msgstr ""
+msgstr "Einspruch gegen diese Inhaltswarnung"
```

Comment:
Not sure if you want to capitalize this in German as well (make it one word), then it would be probably "Inhaltswarnungseinspruch". But your translations works as well.

Reply:
Thanks, I'll go with your suggestion 👍